marklogic-data-hub/src/main/resources/ml-modules/root/com.marklogic.hub/lib/flow-lib.xqy

```diff
@@ -612,16 +612,32 @@ declare function flow:validate-entities()
     let $_ :=
       for $plugin in $flow/hub:plugins/hub:plugin
       let $destination := $plugin/@dest
+      let $module-uri := $plugin/@module
+      let $module-name := hul:get-module-name($module-uri)
+      let $filename as xs:string := hul:get-file-from-uri($module-uri)
+      let $type := flow:get-type($filename)
+      let $ns :=
+        if ($type eq $flow:TYPE-JAVASCRIPT) then ()
+        else
+          $flow:PLUGIN-NS || fn:lower-case($module-name)
+      let $func-name :=
+        if ($type eq $flow:TYPE-JAVASCRIPT) then
+          "create" || functx:capitalize-first($destination)
+        else
+          "create-" || $destination
       return
         try {
-          flow:run-plugin(
-            $plugin,
-            $data-format,
-            "123",
-            (),
-            (),
-            (),
-            $options)
+          if ($type eq $flow:TYPE-XQUERY) then
```

Comment:
I think this section needs a comment or two for the developer who looks at this somewhere down the road. State what kind of errors are being looked for, note that the JS check won't find much, and mention that the open nature of the input prevents more interesting checks.